### ISSUE TITLE
Use same emoji groups as Telegram app

### ISSUE THREAD

**Issue by PanderMusubi** [telegramdesktop/tdesktop#6398]
### Steps to reproduce
1. Search an emoji in Telegram desktop, e.g. checkered finish flag
2. Search that emoji too on mobile device in Telegram and WhatsApp app
3. Notice the difference in emoji groups, emoji group names and merging of two emoji groups

### Expected behaviour
Grouping of emoji should be identical to that of Telegram app and WhatsApp app or desktop.

Telegram app and WhatsApp app & desktop have these eight named groups:
https://github.com/DrKLO/Telegram/blob/master/TMessagesProj/src/main/res/values/strings.xml#L779

    <string name="Emoji1">Smileys and people</string>
    <string name="Emoji2">Animals and nature</string>
    <string name="Emoji3">Food and drink</string>
    <string name="Emoji4">Activity</string>
    <string name="Emoji5">Travel and places</string>
    <string name="Emoji6">Objects</string>
    <string name="Emoji7">Symbols</string>
    <string name="Emoji8">Flags</string>

### Actual behaviour
Group names are not identical to Telegram app and the last groups are joined.

This can be seen in:
https://github.com/telegramdesktop/tdesktop/blob/dev/Telegram/Resources/langs/lang.strings#L1138

    "lng_emoji_category1" = "People";
    "lng_emoji_category2" = "Nature";
    "lng_emoji_category3" = "Food & Drink";
    "lng_emoji_category4" = "Activity";
    "lng_emoji_category5" = "Travel & Places";
    "lng_emoji_category6" = "Objects";
    "lng_emoji_category7" = "Symbols & Flags";

Note that these groups are named differently in the desktop source file https://github.com/telegramdesktop/tdesktop/blob/dev/Telegram/SourceFiles/codegen/emoji/generator.cpp#L432 so that might need consolidation too.

### Configuration
**Operating system:**
n/a

**Version of Telegram Desktop:**
1.15.11

**Used theme**:
n/a

**Comment by yashpalgoyal1304** [comment]
I would suggest taking some _inspiration_ from window 10's emoji panel's grouping too. one reason is that it would make implementation of https://github.com/telegramdesktop/tdesktop/issues/8283#issuecomment-663343020 much easier as it separates the group `People`  from `Smiley faces and animals`.

Window 10 emoji groups for reference:
Note that some tabs like `Flags` are missing though (bad).

    Smiley faces and animals
    People
    Celebrations and objects
    Food and plants
    Transportation and places
    Symbols

**Comment by yashpalgoyal1304** [comment]
I would suggest splitting `Smileys` and `People` for a better implementation of https://github.com/telegramdesktop/tdesktop/issues/8283#issuecomment-.

<details>
<summary>Ref: Window 10 emoji panel's groups:</summary>

    Smiley faces and animals
    People
    Celebrations and objects
    Food and plants
    Transportation and places
    Symbols

</details>

So, final list i would like would be the following. Grouping is mainly following TApp (expected section), and the naming is following TDesk (short and nice).

    Smileys
    People
    Nature
    Food & Drink
    Activity
    Travel & Places
    Objects
    Symbols
    Flags

**Comment by PanderMusubi** [comment]
No problem with it if it helps that other issue.

**Comment by yashpalgoyal1304** [comment]
just mentioning this grouping here: https://unicode-table.com/en/emoji/#link-objects \
lol, that is wayy tooo refined.

**Comment by PanderMusubi** [comment]
Yes, those subcategories are one level too much. Think for usability it is good to have the (almost) same categories as the official top-level categorization and what other popular instant messengers use (i.e. WhatsApp). Looking forward to the PR ;-)

**Comment by yashpalgoyal1304** [comment]
i havent got time to do a comparison for that yet, so, just wondering have u compared the sort (i.e. without the subgrouping)??. 

PR from me!! lol, i aint a programmer unfortunately 😅

**Comment by PanderMusubi** [comment]
Please keep this issue open until solved.
